contracts/src/tokens/lock/LockBase.sol

```diff
@@ -36,10 +32,6 @@ abstract contract LockBase is ILockBase {
   function _disableLock(address caller) internal {
     LockStorage.Layout storage ds = LockStorage.layout();
 
-    if (ds.enabledByAddress[caller] == false) {
```

Comment:
Why are we removing this?

Reply:
Can leave it. But there's no code path using it.